packages/x-adapter-platform/src/schemas/models/facet.schema.ts

```diff
@@ -20,29 +16,12 @@ export const facetSchema = createMutableSchema<
 >({
   id: 'facet',
   label: 'facet',
-  modelName: source => {
-    return getFacetConfig(source.facet).modelName as any;
-  },
+  modelName: ({ type }) => getFacetConfig(type).modelName as any,
   filters: {
     $path: 'values',
-    $subSchema: ({ facet }) => getFacetConfig(facet).schema as any,
+    $subSchema: ({ type }) => getFacetConfig(type).schema as any,
     $context: {
       facetId: 'facet'
     }
   }
 });
-
-export const facetsConfig: FacetsConfig = {
```

Comment:
With this refactor I see unnecessary to keep this object, but I'm not sure if it could be used for some custom cases. Let me know if we should keep it or not 🙏